plugins/modules/docker_host_info.py

```diff
@@ -126,6 +126,15 @@
     disk_usage: yes
   register: result
 
+- name: Get info on docker host and list containers matching the filter
```

Comment:
You should also update the documentation for `containers_filters` and the other `_filters` options.